clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -582,15 +651,19 @@ public void initStore(final Store<?, ?> resource) {
       clusteredStore.storeProxy.addInvalidationListener(new ServerStoreProxy.InvalidationListener() {
         @Override
         public void onInvalidateHash(long hash) {
+          Enum result = StoreOperationOutcomes.EvictionOutcome.SUCCESS;
+          clusteredStore.evictionObserver.begin();
```

Comment:
hmm... let me double check that in a test.  I actually didn't add that code.  Not that I don't trust you but I also need to do some clustered eviction testing so this will force me to do it now.  Where can I create a clustered test that uses management stats?  I tried in :clustered:client but it doesn't have a management dependency.